scripts/lib/lando-extras.js

```diff
@@ -0,0 +1,219 @@
+const yaml = require("yaml-js");
+const fs = require("fs");
+
+class LandoExtras {
+    #ymlPath;
+    #localConfigFile;
+    #extrasConfigFile;
+    #extras = [];
+    #localConfig = {};
+
+    /**
+     * Setup LandoExtras.
```

Comment:
noun should be a verb
```suggestion
     * Set Up LandoExtras.
```